libs/hdf-converters/src/conveyor-mapper.ts

```diff
@@ -0,0 +1,244 @@
+import {ExecJSON} from 'inspecjs';
+import _ from 'lodash';
+import {version as HeimdallToolsVersion} from '../package.json';
+import {BaseConverter, ILookupPath, MappedTransform} from './base-converter';
+const DEFAULT_NIST_TAG = ['SI-2', 'RA-5'];
+
+function createDescription(
```

Comment:
probably worthwhile to reorder functions so that related functions are closer together.